snuba/datasets/plans/single_storage.py

```diff
@@ -150,6 +152,7 @@ def build_plan(self, request: Request) -> ClickhouseQueryPlan:
                 db_query_processors=[
                     *storage.get_query_processors(),
                     *self.__post_processors,
+                    MandatoryConditionApplier(),
```

Comment:
There is another place where the ClickhouseQueryPlan is built above in the SingleStorageQueryPlanBuilder. Please add it there as well.